### PR TITLE
src: modify to use nodeshift instead of oc

### PR DESCRIPTION
**Pull request by helio-frota**
related to #24

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -34,4 +34,19 @@ Execute bonjour locally with Docker
 3. The application will be running at: <http://localhost:8080/api/bonjour>
 
 
+Run bonjour in OpenShift
```
Comment (lance) [inline Readme.md]: @helio-frota I believe that the Readme.md should remain unchanged, so this should be reverted.

**Comment by rafabene** [comment]
Do you plan to also send changes to https://github.com/redhat-helloworld-msa/helloworld-msa ?

**Comment by helio-frota** [comment]
@rafabene not yet, we need to replace from `oc commands ` to `npm run openshift` ?

**Comment by rafabene** [comment]
NO replacement is expected, we need just to ADD nodeshift as an option#2 to deploy this service. Example: https://github.com/redhat-helloworld-msa/helloworld-msa/blob/master/aloha.adoc#option-2-deploy-project-via-fabric8-maven-plugin

We need also to update:
- Probe path here: https://github.com/redhat-helloworld-msa/helloworld-msa/blob/master/bonjour.adoc#enable-readiness-probe and here https://github.com/redhat-helloworld-msa/helloworld-msa/blob/master/ansible/vars.yml#L38

**Comment by helio-frota** [comment]
Closing this PR. 
It looks like I started doing other things at that time. :+1:

**Pull request closed without merging**